src/auth/AuthSessionHandler.cc

```diff
@@ -38,14 +38,3 @@ AuthSessionHandler *get_auth_session_handler(CephContext *cct, int protocol, Cry
   }
   return NULL;
 }
-
-
-void AuthSessionHandler::print_auth_session_handler_stats() {
```

Comment:
@tchaikov If we remove them, then it would remove this purpose too. Basically, I'm not in favour of removing all the codes that are unused, as it may be useful later. What do you think ?

`  // Keep stats on how many messages were signed, how many messages were encrypted, how many  signatures were properly checked, and how many messages were decrypted. PLR`